.github/workflows/main.yml

```diff
@@ -70,6 +70,11 @@ jobs:
             echo "package.json not found, skipping yarn commands."
           fi
 
+      - name: Check formatting
+        run: |
+          yarn format --check --no-write
```

Comment:
This reduces to 

`./arches/app/media/node_modules/.bin/prettier ./arches/app/src --write --check --no-write`

The `--write --no-write` isn't ideal. Is it worth taking the `--write` flag out of `yarn format`? Or adding a second yarn command to `package.json` for exclusively checking?

Reply:
Good catch. If I go with `prettier:check` and `prettier:fix` it will rhyme with the eslint commands.